src/main/kotlin/io/github/nomisrev/service/ArticleService.kt

```diff
@@ -18,9 +19,18 @@ data class CreateArticle(
   val tags: Set<String>
 )
 
+data class GetFeed(
+  val userId: UserId,
+  val limit: Long,
+  val offset: Long,
+)
+
 interface ArticleService {
   /** Creates a new article and returns the resulting Article */
   suspend fun createArticle(input: CreateArticle): Either<DomainError, Article>
+
+  /** Get the user's feed which contains articles of the authors the user followed */
+  suspend fun getUserFeed(input: GetFeed): Either<DomainError, MultipleArticlesResponse>
```

Comment:
Same as `ArticlePersistence`

```suggestion
  suspend fun getUserFeed(input: GetFeed): MultipleArticlesResponse
```